test/features/parsoid/transform.js

```diff
@@ -3,6 +3,7 @@
 const assert = require('../../utils/assert.js');
```

Comment:
Can you try reimporting 'assert' from api-testing too and use chai asserts instead of this?

e.g. const { REST, assert } = require('api-testing') and then your 'assert' will be a reexport of [chai library](https://www.chaijs.com/api/assert/). Then you can change some of the asserts to use the lib, which is more powerful

Reply:
Thanks @Pchelolo, I have reimported `assert` from api-testing and used chai asserts as advised.